experimental/MolecularBiology/WesternBlot/README.md

```diff
@@ -1 +1,108 @@
 
+# Protein Gel and Western Blot
+
+## reagents stored at the protein gel bench, JSCBB B450
+- 1.5M Tris pH 8.8
+- 1M Tris pH 6.8 
+- Loading buffer
+- Tris glycine SDS running buffer
+- Tris glycine methanol transfer buffer
```

Comment:
Are these premade buffers? If so can we add catalog number?
Or the recipe -- I know it seems minor, but that is a good detail for reproducibility -- just being extra detailed -- sorry :)